modules.d/90lvm-systemd/pvscan-udev-initrd.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/sh
```

Comment:
```suggestion
#!/bin/bash
```

If you want to use arrays.